features/guidance/dedicated-turn-roads.feature

```diff
@@ -165,8 +165,8 @@ Feature: Slipways and Dedicated Turn Lanes
             | qe    | secondary_link | Ettlinger Allee    |      | yes    |
 
         When I route I should get
-            | waypoints | route                                                     | turns                    |
-            | a,o       | Schwarzwaldstrasse (L561),Ettlinger Allee,Ettlinger Allee | depart,turn right,arrive |
+            | waypoints | route                                                    | turns                    | ref        |
+            | a,o       | Schwarzwaldstrasse,Ettlinger Allee,Ettlinger Allee       | depart,turn right,arrive | L561,L561, |
```

Comment:
This is actually an existing bug where in post-processing we don't properly update collapsed steps with the correct destination and pronunciation data along with name. PR coming for this fix, which can then include moving `ref` in preprocessing as well.